strawberryfields/apps/similarity.py

```diff
@@ -510,21 +652,98 @@ def feature_vector_mc(
         loss (float): fraction of photons lost in GBS

```

Comment:
On line above, would also mention that the samples are optional like I suggested in `feature_vector_orbits`.